<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditLocations</class>
 <widget class="QWidget" name="EditLocations">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Locations</string>
  </property>
  <widget class="QPushButton" name="addToList">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>220</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Add To List</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="locationName">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>80</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Location/ Shop name</string>
   </property>
  </widget>
  <widget class="QListWidget" name="locationList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>11</y>
     <width>261</width>
     <height>511</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="removeButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>261</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="address">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>130</y>
     <width>331</width>
     <height>71</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Address</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>270</y>
     <width>329</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Address:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAddress">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>330</y>
     <width>329</width>
     <height>221</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
